readme.md

```diff
@@ -9,34 +9,39 @@
 Deliberation is the topmost layer in a robotics architecture sometimes also called mission or planning layer.
 It aims at controlling the robots behavior towards its extended goal or function.
 This includes pre-programmed state machines, automated symbolic planning as well as behavioral reaction to unforeseen situations happening at runtime.
+For an introduction to robot deliberation, refer to [Ingrand and Ghallab (2017)](https://hal.science/hal-01137921).
 
 ## Contents
 
-- [Packages](#packages)
+- [Software Packages](#software-packages)
 - [Presentations](#presentations)
 - [Papers and Book Chapters](#papers-and-book-chapters)
 - [Blog Posts](#blog-posts)
 - [Podcasts](#podcasts)
 - [Demos](#demos)
 
-## Packages
+## Software Packages
```

Comment:
@ct2034 This further opens up the opportunity for a formal verification / model checking category in the event that you want to add some of the CONVINCE toolchain into this list.

Reply:
Thanks. This sounds like a great idea